app/src/ui/diff/diff-line-gutter.tsx

```diff
@@ -100,34 +127,26 @@ export class DiffLineGutter extends React.Component<IDiffGutterProps, {}> {
    * Indicate whether the current gutter element is selected
    */
   public isIncluded(): boolean {
```

Comment:
Should this be `isSelected`?

Reply:
Yeah, it probably should be. Our terminology here is all over the place but I'm not keen to address that in this PR.

Reply:
I brought it up because all of the uses of the method assign its result to something including `selected`.